lib/main.js

```diff
@@ -18,6 +18,7 @@ const DEFAULT_MAX_AGE = 1209600;
  *   authenticates the client token.
  * @param {object} [options.defaults={}] - Token issuance defaults.
  * @param {object} [options.logger] - Logger.
+ * @param {object} [options.cache] - Cache.
```

Comment:
This is optional in the docs here, but it looks required both as a breaking change in the changelog -- and in code usage below. Should this be optional and a minor release ... or should this be required and a breaking release?

Reply:
Marking as required, since it is a security feature.